src/components/Eliza/ElizaIntegrationLayer.tsx

```diff
@@ -211,6 +212,7 @@ export const ElizaIntegrationLayer: React.FC<ElizaIntegrationLayerProps> = ({
         getAgentDeploymentStatus={getAgentDeploymentStatus}
         ensureUserSubscription={ensureUserSubscription}
         projectId={activeProjectId}
+        triggerTrackingEvent={trackCustomEvent}
```

Comment:
I'm yet to read all the source, but at the moment noticed three different names for capturing events. Likely that there's a reason for that, but at very least we can normalize/standardized the main property name, e.g. `captureEvent `. The name `captureEvent ` is sounder.

```suggestion
        captureEvent={trackCustomEvent}
```

I'd go further, looking into why there are different utility methods with different names. Could we not have a single method which computes any needs in its body business logic? 

At risk of missing something here, I'd suggest having a conventional utility method to address these requirements.

Reply:
just standardized the event capturing method to "captureEvent". thanks!